.readthedocs.yaml

```diff
@@ -4,19 +4,18 @@
 version: 2
 
 build:
-  os: ubuntu-20.04
+  os: "ubuntu-20.04"
   tools:
-    python: "3.10"
+    python: "mambaforge-22.9"
```

Comment:
Does this give us a an environment with python 3.10? Might be worth pointing to the mambaforge docs as a comment or something. The `22.9` refers to `mamba`'s version? Or a version of Python that comes from `mambaforge`?

Reply:
This gives us an environment fully managed by conda, so we can specify whatever we want in `doc/environment.yml`. I was thinking we should let the other specifications determine the python version so it will auto-upgrade. Thoughts?

Reply:
Got it, that makes sense to me. lets add a comment here with a note that this installs mamba/conda and that the python version used by RTD will be determined by what's pinned in the `environment.yml`.

Reply:
Good call, that's all pretty well-documented here so I'll also include this link:

https://docs.readthedocs.io/en/stable/guides/conda.html

Reply:
166a0e19